includes/blocks/class-sensei-course-outline-block.php

```diff
@@ -57,41 +57,22 @@ class Sensei_Course_Outline_Block {
 	 * Sensei_Course_Outline_Block constructor.
 	 */
 	public function __construct() {
-
-		add_action( 'init', [ $this, 'register_course_template' ], 101 );
-		add_action( 'init', [ $this, 'register_blocks' ] );
-		add_action( 'init', [ $this, 'init' ] );
-
 		$this->course = new Sensei_Course_Outline_Course_Block();
 		$this->lesson = new Sensei_Course_Outline_Lesson_Block();
 		$this->module = new Sensei_Course_Outline_Module_Block();
+
+		$this->register_blocks();
 	}
 
 	/**
-	 * Initialize block instance.
+	 * Resets block content.
```

Comment:
Should we rename the method too to match the described purpose?

Reply:
I didn't do that because it is a public method and we should deprecate it first. But truth is that there is practically no chance anybody using it 1ebae7abff101d64f8dbf23373eed1d6c65dbeaf